sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -221,6 +223,25 @@ private static void unwrapUserCodeException(UserCodeException e) throws Exceptio
    * been finished
    */
   public void processElement(InputT element) throws Exception {
+    processTimestampedElement(TimestampedValue.of(element, null));
```

Comment:
`null` -> `BoundedWindow.TIMESTAMP_MIN_VALUE`, or add a default constructor for the same.